### DISCUSSION TITLE
Do more than one operation on a resource simultaneously

### DISCUSSION THREAD

**Discussion by fvizcay** [frePPLe/frepple#499] [Q&A]
Hi,

I need to know if it is possible to have a resource that performs different operations simultaneously.

Ex: A camera where different items are placed at the same time for a certain time.

Thanks!

**Comment by jdetaeye** [comment]
Of course it's possible.

You have the "resource.size" which defines the resource capacity.
And you have the "operationresource.quantity" field which defines how much of the resource capacity is needed to perform the operation.
Eg resource.size = 5, and operationresource.quantity = 1   ===> this machine can do 5 of those operations in parallel

**Reply by fvizcay** [reply]
Johan,

Thank you very much for your quick response.

Maybe I didn't explain the case very well. I meant to perform different operations simultaneously in one resource.

This would be the case.

I have 3 different items that are in state 1:
-Item A (state 1)
-Item B (state 1)
-Item C (state 1)

The 3 items are introduced at the same time in a chamber (resource) that after X time, transforms them to state 2.

After this time X, we obtain:
-Item A (state 2)
-Item B (state 2)
-Item C (state 2)

*1 Item A (state 1) is consumed to produce 1 Item A (state 2)., etc.

Thank you very much for your help.
